src/main.ts

```diff
@@ -185,11 +199,15 @@ async function run(): Promise<void> {
   }
 }
 async function closePullRequest(number: number) {
-  await client.rest.pulls.update({
-    ...context.repo,
-    pull_number: number,
-    state: 'closed'
-  })
+  try {
+    await client.rest.pulls.update({
+      ...context.repo,
+      pull_number: number,
+      state: 'closed'
+    })
+  } catch (error) {
+    if (error instanceof Error) throw new Error(errors.closingPullRequest)
```

Comment:
I think we'd want to somehow log the original `error` regardless.

Reply:
new commit just dropped - still haven't smoke tested, buyer beware